### PR TITLE
Basic Status Grid

### PR DESCRIPTION
**Pull request by CaseyHillers**
Scrolls both horizontally and vertically. However, the vertical scroll is much slower than the horizontal scroll (SingleChildScrollView is quicker than the GridView scroll).

### CODE REVIEW

File: app_flutter/lib/status_grid.dart [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'result_box.dart';
+
```
Comment (jonahwilliams) [inline app_flutter/lib/status_grid.dart]: Add a short sentence describing what this is for

File: app_flutter/lib/status_grid.dart [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+class StatusGrid extends StatelessWidget {
+  static const int taskCount = 80; // rough estimate based on existing dashboard
```
Comment (jonahwilliams) [inline app_flutter/lib/status_grid.dart]: Generally good practice to add a const constructor that takes a key:

`const StatusGrid({Key key}) : super(key: key);`

File: app_flutter/lib/status_grid.dart [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'result_box.dart';
+
+class StatusGrid extends StatelessWidget {
```
Comment (jonahwilliams) [inline app_flutter/lib/status_grid.dart]: Good to add a basic smoke test. It lowers the barrier for new tests in the future.

File: app_flutter/lib/status_grid.dart (hunk 1 above)
Comment (CaseyHillers) [reply app_flutter/lib/status_grid.dart]: Added!

File: app_flutter/lib/status_grid.dart (hunk 2 above)
Comment (CaseyHillers) [reply app_flutter/lib/status_grid.dart]: Added!

File: app_flutter/lib/status_grid.dart (hunk 3 above)
Comment (CaseyHillers) [reply app_flutter/lib/status_grid.dart]: Added! Left a TODO to remove the smoke test when useful tests are added.

**Pull request merged**